docs/website/contents/explanation/consensus-protocol.md

```diff
@@ -0,0 +1,267 @@
+    - `RestrictedVRFTiebreaker`: This flavor restricts the VRF tiebreaker comparison to situations where the slot numbers of the chain tips differ by at most a specified maximum distance (`maxDist`).
+The primary motivation for this restriction is to favor blocks that were diffused earlier (in earlier slots) over those diffused later, even if the later block has a "better" VRF tiebreaker value. This aims to mitigate [issues](https://github.com/IntersectMBO/ouroboros-network/issues/2913) caused by poorly configured or resource-constrained pools that might diffuse blocks later.
+
+### On the Transitivity of Praos Chain Ordering
+
+Praos chain ordering is **not transitive**, regardless of the VRF tiebreaker flavor.
+
+Consider the following select views where chains `A`, `B`, and `C` have the same length:
+
+|                 | A | B | C |
+|-----------------|---|---|---|
+| Issuer          | x | y | x |
+| Slot            | 0 | i | 0 |
+| `opcert` number | 2 | j | 1 |
+| VRF             | 3 | 2 | 1 |
+
+Lower-case letters stand for arbitrary values (two letters designate the same value if and only if they are same letter).
+
+In this example we have:
+
+- `B` is preferred over `A`, since `B` has lower VRF than `A`.
+- `C` is preferred over `B`, since `C` has lower VRF than `B`.
+- However `C` is **not** preferred over `A`, since they have the same issuer and slot, and therefore we prefer the chain with the highest `opcert` number (2), therefore `A` is preferred over `C`.
+
+Also, the `RestrictedVRFTiebreaker` flavour breaks the transitivity of chain ordering. To see this consider the following example where chains `D`, `E`, and `F` have the same length and different issuers, and assume `maxDist = 5` slots:
+
+|              | D | E | F |
+| ------------ | - | - | - |
+| Slot         | 0 | 3 | 6 |
+| VRF          | 3 | 2 | 1 |
+
+
+We have that:
+- `E` is preferred over `D`, since `E` has lower VRF than `D` and `|0 - 3| < 5`.
+- `F` is preferred over `E`, since `F` has lower VRF than `E` and `|3 - 6| < 5`
+- However, `D` is **not** preferred over `F`, but instead `D` and `F` are equally preferred since `|0 - 6| > 5` which implies that the VRF values of `D` and `F` are not used.
+
+Despite the non-transitivity, the fundamental Consensus properties, such as [Common Prefix](TODO-ref!), are not affected. This is because the primary factor for chain selection for a [caught-up](TODO-ref!) node remains the chain length, with longer chains always being preferred. However, a non-transitive chain ordering brings the following complications:
+
+- **Implementation**: The use of `sortBy` from base in `chain` selection, which is typically expected to work with transitive relations, could become a concern. While preliminary property tests suggest it works for the current non-transitive order, there's a theoretical risk that future GHC implementations might interact non-trivially.
```

Comment:
from base -> from [`base`](https://hackage.haskell.org/package/base)

theoretical -> potential

interact non-trivially -> behave in surprising ways

Reply:
Thanks. For the last suggestion, I used "might behave in unexpected ways". Let me know if you prefer your version.